src/backend/catalog/aclchk.c

```diff
@@ -3935,6 +3972,10 @@ pg_extprotocol_aclcheck(Oid ptcid, Oid roleid, AclMode mode)
 AclResult
 pg_filesystem_aclcheck(Oid fsysid, Oid roleid, AclMode mode)
 {
+	/* Bypass all permission checking on QE. */
+	if (GP_ROLE_EXECUTE == Gp_role)
+		return ACLCHECK_OK;
+
```

Comment:
@wcl14 if you use vim, try it:
`set tabstop=2
set softtabstop=2
set expandtab`